src/anyio/_backends/_asyncio.py

```diff
@@ -485,6 +493,8 @@ async def spawn(self, func: Callable[..., Coroutine], *args, name=None) -> None:
         _task_states[task] = TaskState(parent_id=id(current_task()), name=name,
                                        cancel_scope=self.cancel_scope)
         self.cancel_scope._tasks.add(task)
+        if self.cancel_scope._cancel_called:
+            create_task(_cancel_soon(task))
```

Comment:
Shouldn't `loop.call_soon()` be used here, and `_cancel_soon()` be made into a regular function?